tabs/pid_tuning.js

```diff
@@ -267,6 +267,34 @@ TABS.pid_tuning.initialize = function (callback) {
             $('.dtermfiltertype').hide();
             $('.antigravity').hide();
         }
+
+        $('input[id="gyroNotch1Enabled"]').change(function() {
+            var checked = $(this).is(':checked');
+            $('.pid_filter input[name="gyroNotch1Frequency"]').val(checked ? DEFAULT.gyro_soft_notch_hz_1 : 0)
+                .attr('disabled', !checked);
+            $('.pid_filter input[name="gyroNotch1Cutoff"]').val($(this).is(':checked') ? DEFAULT.gyro_soft_notch_cutoff_1 : 0)
```

Comment:
@basdelfos Is there a reason for not using the `checked` variable on this line? Yet line 273, 281, 283, 289, 291 do use that variable?

Reply:
No reason other than that I probably missed a replace in line 275 after introducing the `checked` variable.

Reply:
I've submitted a patch to make this consistent https://github.com/betaflight/betaflight-configurator/pull/659